README.md

```diff
@@ -17,6 +17,18 @@ But if you want to **contribute** and make this much better for other developer
 
 If you created something awesome and want to contribute then feel free to open an [pull request](https://github.com/saadpasta/developerFolio/pulls).
 
+## Table of Contents
+- [Sections](#sections)\
+- [Getting Started](#getting-started)\
+- [How to Use](#how-to-use)\
+- [Github Setup](#github-setup-for-open-source-projects)\
+- [Change and Customize](#change-and-customize-every-section-according-to-your-need)\
+- [Deployment](#deployment)\
+- [Technologies Used](#technologies-used)\
+- [Illustrations](#illustrations)\
+- [For the Future](#for-the-future)\
+- [Project Maintainers](#project-maintainers)\
```

Comment:
Let's remove this and just keep `Contributors` that is linked to `#project-maintainers` to avoid differentiating between all contributors!